src/main/java/org/opensearch/knn/plugin/transport/UpdateModelGraveyardRequest.java

```diff
@@ -18,7 +18,7 @@
 /**
  * Request for updating model graveyard while processing delete model request
  */
-public class UpdateModelGraveyardRequest extends AcknowledgedRequest<UpdateModelGraveyardRequest> {
+public class UpdateModelGraveyardRequest extends ClusterInfoRequest<UpdateModelGraveyardRequest> {
```

Comment:
why do we need to switch here? Cant we just keep the acked request and throw exception?

Reply:
I think that was one of the things I was testing to try to get everything to work.  It's not necessary, I'll remove it